Bio/SeqIO/SeqXmlIO.py

```diff
@@ -519,7 +524,7 @@ def _write_species(self, record):
                     local_ncbi_taxid = None
                 else:
                     ValueError(
-                        'Multiple entries for record.annotations["ncbi_taxid"], %r'
+                        "Multiple entries for record.annotations['ncbi_taxid'], %r"
```

Comment:
You didn't need to change this line, maybe put it back as it is a change to the exception string someone might be catching?